docs/install/install-edex.md

```diff
@@ -51,45 +39,27 @@ sudo ./awips_install.sh --edex
 
 !!! note "**awips_install.sh --edex** will perform the following steps (it's always a good idea to review downloaded shell scripts):"
 
-       1. Saves the appropriate Yum repo file to `/etc/yum.repos.d/awips2.repo`
-       2. Increases process and file limits for the the *awips* account in `/etc/security/limits.conf`
-       3. Creates `/awips2/data_store` if it does not exist already
-       4. Runs `yum groupinstall awips2-server`
-       5. Attempts to configure the EDEX hostname defined in `/awips2/edex/bin/setup.env`
-       6. Alerts the user if the *awips* account does not exist (the RPMs will still install)
+       1. Checks to see if EDEX is currently running, if so stops the processes `edex stop`
+       2. If EDEX is installed, asks the user if it can be removed and where to backup data to and does a `yum groupremove awips2-server`
+       3. If the user/group awips:fxalpha doesn't exist, it will be created
```

Comment:
keeping with the same tense... "...doesn't exits, it gets created" or "it creates it"